File: 02_ProgrammingFundamentals/_04_Methods/_01_Lecture/_02_Grades.java
package _02_ProgrammingFundamentals._04_Methods._01_Lecture;

import java.util.Scanner;

public class _02_Grades {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        double grade = Double.parseDouble(scan.nextLine());

        // Create method to qualify grades
        whatIsTheGrade(grade);

        scan.close();
    }

    private static void whatIsTheGrade(double grade) {
        if (grade >= 2.00 && grade < 3.00) {
            System.out.println("Fail");
        } else if (grade < 3.50) {
            System.out.println("Poor");
        } else if (grade < 4.50) {
            System.out.println("Good");
        } else if (grade < 5.50) {
            System.out.println("Very good");
        } else if (grade < 6.00) {
            System.out.println("Excellent");
        }
    }
}

File: 24_C#Fundamentals/_04_Methods/_01_Lecture/_02_Grades/_02_Grades.cs
using System;

public class _02_Grades
{
    public static void Main(string[] args)
    {
        // Read input value
        double grade = double.Parse(Console.ReadLine());

        // Create method to qualify grades
        whatIsTheGrade(grade);
    }

    private static void whatIsTheGrade(double grade)
    {
        if (grade >= 2.00 && grade < 3.00)
        {
            Console.WriteLine("Fail");
        }
        else if (grade < 3.50)
        {
            Console.WriteLine("Poor");
        }
        else if (grade < 4.50)
        {
            Console.WriteLine("Good");
        }
        else if (grade < 5.50)
        {
            Console.WriteLine("Very good");
        }
        else if (grade < 6.00)
        {
            Console.WriteLine("Excellent");
        }
    }
}
